<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>711</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>搜索酒店</string>
     </attribute>
     <widget class="QTableWidget" name="table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>741</width>
        <height>331</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearch">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>300</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>28</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineSearchCity">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>搜索城市</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>搜索您中意的一栖之地</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnDefault">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>默认排序</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnComment">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>按评价好坏排序</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnPrice">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>按价格高低排序</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExit">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>500</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>保存并退出登录</string>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>50</y>
        <width>311</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineDays">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>180</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>100</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>请输入入住天数</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>请选择入住日期</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDetail">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>400</y>
        <width>221</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>查看详细信息</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineSearchDis">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>搜索地区（可不填）</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>所有房型</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>钟点房</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>双人房</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>大床房</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>奢华房</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>总统套房</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>订单列表</string>
     </attribute>
     <widget class="QTableWidget" name="table2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>791</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>241</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>36</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>订单列表</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPay">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>210</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>确认付款</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRefund">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>330</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>申请退款</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnComment">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>450</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>评价订单</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRefresh">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>90</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>刷新列表</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>信息维护</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>941</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>36</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>一栖之地——酒店预订管理系统</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>150</y>
        <width>571</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>36</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>C++程序设计与训练</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>350</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入手机验证码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnChangePass">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>500</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>确认更改</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineOldPassword">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>400</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入旧密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineNewPasss">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>450</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>请输入新密码</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>350</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>获取手机验证码</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>300</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>更改密码</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
